assets/js/googlesitekit/datastore/site/registry-key.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * File Contents Description.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+
+// Actions
+const SET_REGISTRY_KEY = 'SET_REGISTRY_KEY';
+
+const INITIAL_STATE = {
+	registryKey: undefined,
+};
+
+export const actions = {
+	/**
+	 * Sets the registryKey in the data store.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {(number|undefined)} registryKey The registryKey for a given store.
```

Comment:
This should just be a string now, since we're using a UUID now, right?

```suggestion
	 * @param {string} registryKey The registryKey for a given store.
```